src/authorizations/components/GenerateTokenDropdown.test.tsx

```diff
@@ -0,0 +1,55 @@
+// Libraries
+import React from 'react'
+import {fireEvent, screen} from '@testing-library/react'
+
+// Fixtures
+import {renderWithReduxAndRouter} from 'src/mockState'
+
+jest.mock('src/shared/constants', () => ({
+  CLOUD: true,
+}))
+
+const isFlagEnabledMock = jest.fn(() => {
+  return false
+})
+jest.mock('src/shared/utils/featureFlag', () => {
+  return {
+    isFlagEnabled: isFlagEnabledMock,
+  }
+})
+
+import GenerateTokenDropdown from 'src/authorizations/components/GenerateTokenDropdown'
+
+const setup = () => {
+  return renderWithReduxAndRouter(<GenerateTokenDropdown />)
+}
+
+describe('GenerateTokenDropdown', () => {
+  it('renders the dropdown', () => {
```

Comment:
follow up: maybe write a test for when `CLOUD` is false so we can be confident this won't accidentally get turned on in OSS

Reply:
Will add this use case in next pr as it is a bit more complicated to mock the `CLOUD` constant to false.